### PR TITLE
Code Review: Issue19: Dashboard: Display outstanding loan.

### PR DESCRIPTION
**Pull request by Ghost---Shadow**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp"
     crossorigin="anonymous">
 
-  <title>Social-Credit</title>
+  <title>React App</title>
```
Comment (saurav-sahu) [inline public/index.html]: Why did you rename it back to `React App`?

File: public/manifest.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "short_name": "Social-Credit",
-  "name": "Get loans in a new way!",
+  "short_name": "React App",
```
Comment (saurav-sahu) [inline public/manifest.json]: Short name should've been `Social-Credit`.

File: public/index.html (hunk 1 above)
Comment (Ghost---Shadow) [reply public/index.html]: Sorry, I did git revert.
